system_tests/speech.py

```diff
@@ -175,3 +185,43 @@ def test_async_recognize_gcs_file(self):
 
         _wait_until_complete(operation)
         self._check_results(operation.results, 2)
+
+    def test_stream_recognize(self):
+        if not Config.USE_GAX:
+            self.skipTest('gRPC is required for Speech Streaming Recognize.')
+
+        with open(AUDIO_FILE, 'rb') as file_obj:
+            for results in self._make_streaming_request(file_obj):
+                self._check_results(results)
+
+    def test_stream_recognize_interim_results(self):
+        if not Config.USE_GAX:
+            self.skipTest('gRPC is required for Speech Streaming Recognize.')
+
+        # These extra words are interim_results that the API returns as it's
+        # deciphering the speech audio. This has a high probability of becoming
+        # out of date and causing the test to fail.
```

Comment:
Nope, it's been highly variable. I've gotten 5 responses and the last time I ran it I got 27...